Add first robotic tag

osanseviero opened the pull request:
For reference: https://twitter.com/theCamusean/status/1575177398852943872 

For projects such as
* https://cliport.github.io/
* https://github.com/NVlabs/6dof-graspnet

Right now the modality is `rl`, but we will rename it to a name that better encompasses both robotics and RL.

![Screenshot from 2022-10-03 23-30-50](https://user-images.githubusercontent.com/7246357/193687842-2c555e8a-ffd1-4604-823b-c0dcc03e5b50.png)
![Screenshot from 2022-10-03 23-30-36](https://user-images.githubusercontent.com/7246357/193687847-9d13ac9a-c72d-4b7c-af99-57754fea1021.png)

natolambert reviewed (approved):
Looks good for an initial solution 🚀

osanseviero commented:
This needs some work (e.g. the icon)  from my side, I'll get to it soon!

julien-c commented:
can we see a screenshot of the icon? =)

mishig25 commented on js/src/lib/components/Icons/IconRobotics.svelte:
```diff
@@ -0,0 +1,22 @@
+	width="1em"
+	height="1em"
+	preserveAspectRatio="xMidYMid meet"
+	viewBox="0 0 32 32"
```
```suggestion
	viewBox="0 0 24 24"
```

Fixes the slight top left tilt in this msg,
Now it fills the entire "square"
<img width="128" alt="image" src="https://user-images.githubusercontent.com/11827707/193679633-bd630de8-3d44-43ab-8384-90697e49b615.png">

gary149 replied:
can we see it in context?

osanseviero replied:
I added screenshots at the top

osanseviero commented:
I added 2 screenshots with a built instance of the Hub. I also fixed the documentation on this as it was outdated and added `robotics` in a couple of other places

julien-c commented:
i think it's too small compared to the other ones, need to crop it a little bit (or change to another icon, but this one is ok IMO) @gary149

osanseviero commented:
Some help in that would be greatly appreciated :)

gary149 commented:
> Some help in that would be greatly appreciated :)

Should be better now.

osanseviero commented:
Thanks!

julien-c commented:
Updated screenshot in context: 
<img width="1056" alt="image" src="https://user-images.githubusercontent.com/326577/194553153-0af1199e-faa1-4009-9097-3198eb3c7c40.png">

osanseviero commented:
Let's ship it! :robot:
